src/DeviceInterface/InterfacePrototype.js

```diff
@@ -236,6 +240,10 @@ class InterfacePrototype {
     }
 
     async startInit () {
+        if (this.alreadyInitialized) return
+
+        this.alreadyInitialized = true
```

Comment:
If you can think of a better way to do this, I'm open to it. Note: it must be here because the rest of the function has `await`s which get scheduled for later for this could run twice. If that happens several tests break so I am not testing this specifically.